src/box/lua/schema.lua

```diff
@@ -2502,6 +2561,26 @@ base_index_mt.alter = function(index, options)
     end
     return box.schema.index.alter(index.space_id, index.id, options)
 end
+base_index_mt.tuple_pos = function(index, tuple)
+    check_index_arg(index, 'tuple_pos')
+    if not tuple then
+        box.error({reason="Usage index:tuple_pos(tuple)"})
```

Comment:
Nit: We use `ER_PROC_LUA` in all other places in this source file to report about usage errors.

Reply:
Done